.gitignore

```diff
@@ -19,3 +19,12 @@ server/testAutomation/
 
 # Include the modules build.gradle file -- this line will override previous exclusion
 !server/modules/build.gradle
+
+# Ignore parts of the .idea dir
+.idea/libraries
+.idea/modules
+.idea/.gitignore
```

Comment:
This is presumably generated by intelliJ.  Interesting that it doesn't think the libraries and modules directories are to be ignored, but whatever.